esp/services/ws_sql/SQL2ECL/SQLExpression.cpp

```diff
@@ -552,7 +552,7 @@ void SQLBinaryExpression::toECLStringTranslateSource(
         }
 
     }
-    else if (translation1.length()<0 && translation2.length()<0)
+    else if (translation1.length() == 0 && translation2.length() == 0)
```

Comment:
Looks good, although calling the "isEmpty()" method is likely somewhat more efficient, especially if in a tight loop

Reply:
This is changing the semantics - was that intended?

Reply:
The original logic was incorrect. The intention here is to return in the case where both operand translations are empty. Otherwise, if an option to ignore a mis-translation is set, we apply some logic. Else we return anyway.